src/vsphere_cpi/spec/unit/cloud/vsphere/drs_rules/drs_lock_spec.rb

```diff
@@ -2,7 +2,8 @@
 require 'timecop'
 
 describe VSphereCloud::DrsLock, fake_logger: true do
-  subject(:drs_lock) { described_class.new(vm_attribute_manager) }
+  subject(:drs_lock) { described_class.new(vm_attribute_manager, drs_lock_name) }
```

Comment:
This is using the default lock_name so you dont need to pass it, in end there is a test to check a custom drs_lock name